package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Facebook SDK support for React Native apps.",
   "main": "lib/commonjs/index.js",
   "module": "lib/module/index.js",
-  "react-native": "src/index.js",
+  "types": "lib/typescript/index.d.ts",
+  "react-native": "src/index.ts",
```

Comment:
Probably will lead to an issue, not everyone is using typescript, I should remove this